filestorage/handlers/dummy.py

```diff
@@ -10,7 +11,9 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         # Store files where the key is the url path and the value is
         # the bytes that were stored to the file.
-        self.files: Dict[str, bytes] = {}
+        self.files: Dict[
+            str, Dict[str, bytes, str, datetime, str, datetime, str, datetime]
```

Comment:
Getting complex enough that a named tuple might be a good idea. I think it's fine as-is though.

Reply:
I updated this to use a named tuple, a lot cleaner.